apps/admin/frontend/src/components/save_backend_file_modal.tsx

```diff
@@ -167,10 +149,11 @@ export function SaveBackendFileModal({
                   options={{
                     // Provide a file name and default to the USB drive's root directory.
                     defaultPath: join(
-                      usbDrivePath ?? '',
+                      usbDriveStatus.mountPoint,
                       basename(defaultRelativePath)
                     ),
                   }}
+                  disabled={!window.kiosk}
```

Comment:
Although the happy path of saving to the USB drive via this modal now _does not_ need `kiosk-browser`, doing "Save As..." still does, as it uses `kiosk-browser` to open the save dialog.

Reply:
Do we want "Save As…" still?

Reply:
I still use it in dev 🤷 but I wouldn't want to maintain it if it will cost us a lot in the future. It may.

The only reason `Save As...` can work is because in Electron, the returned `File` object from the system dialog has a path. In a regular browser, this doesn't work, because security. So if we are aiming to drop `kiosk-browser`, the replacement would either need to support this or we'd need to remove this.